src/test/java/com/qa/automation/runners/TestRunner.java

```diff
@@ -8,10 +8,12 @@
 @CucumberOptions(
         features="classpath:features",//to tell cucumber where is ur feature file
         glue="com.qa.automation.stepdefs", // to tell cucumber where is ur step def code
-        tags="@ProdMainCategory", // to tell which tagged feature file to execute
```

Comment:
Please remove the tags, the "tags" should be empty.

Reply:
Made the changes please check and approve the PR

Reply:
It Seems like the changes are good for now, approved.